src/Services/GTT/shared/GTT.Application/Utils/Constants.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GTT.Application.Utils
+{
+    public class Constants
+    {
+        public class AzureStorageImage
+        {
+            public const string Type = "image";
+            public const int Size = 10;
```

Comment:
Size should be more descriptive, change to UploadedImageFileSize
Same to Type, if Type is for valid uploaded image file types, than change the variable name to : AllowUploadedImageFileTypes